cfme/containers/image.py

```diff
@@ -152,6 +153,81 @@ def all(self):
                                            provider=provider or get_crud_by_name(ems_name)))
         return images
 
+    def check_compliance_multiple_images(self, image_entitys, wait_for_finish=True, timeout=240):
+        """Initiates compliance check and waits for it to finish."""
+
+        # Chose Check Compliance of Last Known Configuration
+        images_view = navigate_to(self, 'All')
+        self.check_image_entitys(image_entitys)
+        wait_for(lambda: images_view.toolbar.policy.is_enabled, num_sec=5,
+                 message='Policy drop down menu is disabled after checking some Images')
+        images_view.toolbar.policy.item_select('Check Compliance of Last Known Configuration',
+                                  handle_alert=True)
+        images_view.flash.assert_no_error()
+
+        # Verify Image summary
+        for image_instance in image_entitys:
+            original_state = 'Never Verified'
+            if wait_for_finish:
+                try:
+                    wait_for(
+                        lambda: image_instance.compliance_status != original_state,
+                        num_sec=timeout, delay=5,
+                        message='compliance state of Image ID, "{}", still matches {}'
+                                .format(image_instance.id, original_state)
+                    )
+                except TimedOutError:
+                    logger.error('compliance state of Image ID, "{}", is {}'
+                                 .format(image_instance.id, image_instance.compliance_status))
+                    raise TimedOutError('Timeout exceeded, Waited too much'
+                                        ' time for check Compliance finish ({}).'.format(timeout))
+
+    def check_image_entitys(self, image_entitys):
```

Comment:
__REQUIRED__
Spelling: image_entitys --> image_entities. please changed both function name and argument and all over the code

Reply:
Done, thank you for this comment. I change this spelling mistake all over the code.